client/me/account/main.jsx

```diff
@@ -46,9 +46,11 @@ import { successNotice, errorNotice } from 'state/notices/actions';
 import { getLanguage, isLocaleVariant, hasTranslationSet } from 'lib/i18n-utils';
 import { isRequestingMissingSites } from 'state/selectors';
 import _user from 'lib/user';
+import { canDisplayCommunityTranslator } from 'components/community-translator/utils';
 
 const user = _user();
 const colorSchemeKey = 'calypso_preferences.colorScheme';
+const enableTranslatorKey = 'enable_translator';
```

Comment:
There is another constant with the same value above in `utils.js`. If they are meant to be the same, we should extract it as a importable constant.

Reply:
Should we add this to the constants file?